pickem/pickem_homepage/live_views.py

```
@@ -0,0 +1,102 @@
+        # Redis unreachable / dropped: end the stream gracefully so the client
+        # falls back to polling, without Django logging an unhandled app error.
+        logger.warning("SSE score stream ended on error", exc_info=True)
+        return
+    finally:
+        # Best-effort cleanup; never raise from finally even if subscribe failed.
+        if pubsub is not None:
+            try:
+                await pubsub.unsubscribe(channel)
+                await pubsub.aclose()
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)